rest_api/actix_web_1/Cargo.toml

```diff
@@ -28,6 +28,7 @@ futures = { version = "0.1", optional = true }
 log = { version = "0.4", optional = true }
 serde = { version = "1", features = ["derive"], optional = true }
 serde_json = { version = "1", optional = true }
+splinter = { path = "../../libsplinter", features = ["rest-api-actix-web-1"], optional = true }
```

Comment:
I'm confused by this - aren't we moving actix-web-1 stuff here?

Reply:
Yes however its not straightforward to move all of it at once. Authorization is a good example, if we move that code we  also have to move the biome credentials and key_management, and all the oauth stuff in one large move to make it coherent. So for the moment libsplinter is a direct dependency but that should be shortlived.